http_service/bugbug_http/models.py

```diff
@@ -61,7 +61,7 @@ def get_model(model_name):
         # Cache the model only if it was last used less than two hours ago.
```

Comment:
This should be handled directly by your class now, or we split the logic in two places.

Reply:
Sounds good, will do.

I did it this way because I was able to abstract the cache as a general purpose 'IdleTTLCache' (as described in its comment) if I separated this logic from the cache. Im not sure if I can come up with as clean of an abstraction if I put all of this functionality in one place.

However, weather or not this idea of mine is 'better' or even 'good' seems very subjective. I'm happy to put the logic into one place and possibly compromise the quality of the abstraction (in my eyes) if you prefer.

Reply:
And I think it's still generic: a cache which expires entries after TTL and which only caches them if they're loaded twice within a TTL timespan. It's just a different kind of cache than just an expiring one :)